app-cpp/app355.h

```diff
@@ -22,139 +22,141 @@
 
 // self
 #include "libsat355.h"
+#include "appPtr.hpp"
 
 // All public interface methods in .hpp files should exist in a named namespace
-namespace app355 {
+namespace app355
+{
 
 #pragma region class OrbitalData
-class OrbitalData
-{
-private:
-    sat355::TLE mTLE;
-    double mLatitude;
-    double mLongitude;
-    double mAltitude;
-    std::string mName;
-    double mMeanMotion;
-
-public:
-    OrbitalData(sat355::TLE inTLE, double inLatitude, double inLongitude, double inAltitude)
-        : mTLE{std::move(inTLE)}, mLatitude(inLatitude), mLongitude(inLongitude), mAltitude(inAltitude)
+    class OrbitalData
     {
-        mName = mTLE.GetName();
-        mMeanMotion = mTLE.GetMeanMotion();
-    }
-
-    OrbitalData() = delete;
-
-    ~OrbitalData() = default;
-
-    const sat355::TLE& GetTLE() const 
-    { 
-        return mTLE; 
-    }
-
-    double GetLatitude() const 
-    { 
-        return mLatitude; 
-    }
-    void SetLatitude(double latitude) 
-    { 
-        mLatitude = latitude; 
-    }
-
-    double GetLongitude() const 
-    { 
-        return mLongitude; 
-    }
-    void SetLongitude(double longitude) 
-    { 
-        mLongitude = longitude; 
-    }
-
-    double GetAltitude() const 
-    { 
-        return mAltitude; 
-    }
-    void SetAltitude(double altitude) 
-    { 
-        mAltitude = altitude; 
-    }
-};
-#pragma endregion {}
+    private:
+        sat355::TLE mTLE;
+        double mLatitude;
+        double mLongitude;
+        double mAltitude;
+        std::string mName;
+        double mMeanMotion;
+
+    public:
+        OrbitalData(sat355::TLE inTLE, double inLatitude, double inLongitude, double inAltitude)
+            : mTLE{std::move(inTLE)}, mLatitude(inLatitude), mLongitude(inLongitude), mAltitude(inAltitude)
+        {
+            mName = mTLE.GetName();
+            mMeanMotion = mTLE.GetMeanMotion();
+        }
+
+        OrbitalData() = delete;
+
+        ~OrbitalData() = default;
+
+        const sat355::TLE &GetTLE() const
```

Comment:
I'm seeing a lot of `type &` notation, instead of `type& ` notation.
i feel `type& ` is preferred because it groups "all the chars involved w/ the type" together.
check your Tools->Editor->CodeStyleSettings to be sure it isn't auto-formatting this way